app/src/main/java/net/skyscanner/backpack/demo/compose/RadioButtonStory.kt

```diff
@@ -98,3 +101,20 @@ fun DisabledCheckedRadioButtonExample() {
     )
   }
 }
+
+@Preview
+@Composable
+fun CustomContentRadioButtonExample() {
+  BackpackPreview {
+    var selected by remember { mutableStateOf(false) }
+    BpkRadioButton(
+      selected = selected,
+      onClick = { selected = !selected },
+    ) {
+      Column {
+        BpkText(text = stringResource(id = R.string.toggle_custom_title), style = BpkTheme.typography.heading5)
```

Comment:
This may mislead used to assume that we actually provide title + subtitle implementation. I'd probably use something like:

Custom
Content